plugins/modules/aci_static_node_inb_mgmt_address.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: aci_static_node_inb_mgmt_address:
+short_descrption: In band management IP address 
+descrption: 
+- Define: Cisco ACI Fabric Node IP address
+Version_added: 
+Options:
+  tenant:
+    descrption:
+    - Name of existing tenant.
+    type: str
+    aliases: [tenant_name]
+  epg:
+    descrption:
+    - The name of the end point group
+    type: str
+    aliases: [epg_name]
+  pod_id:
+    descrption:
+    - The pod number part of the tDN
+    - pod_id is usually an integer 
+    type: int
+  node_name: 
+    descrption
+    - ACI Fabric node names
+    type: str  
+  node_id:
+    descrption:
+    - ACI Fabric node node_id
+    type: str
+  band_type:
+    descrption:
+    - Out of Band for Nodes
+    type: str 
+  ipv4_address:
+    descrption:
+    - ipv4 address for inb mgmt 
+    type: str
+  ipv4_gw:
+    descrption:
+    - GW address for inb mgmt
+    type: str 
+  ipv6_address:
+    descrption:
+    -  ipv6 address for inb mgmt
+    type: str
+  state:
+    descrption:
+    - Use pesent or absent for adding and removing
+    - Use query for listing an object or multiple ojbects
+    type: str
+    choices: [present, absent, query]
+extends_documentation_fragment: aci
+note:
+- The tenant mgmt , epg default, pood_id, node_id, node_name should exist befor using this module
```

Comment:
It would be good to specify the modules that create the said attributes.

pod_id